### PR TITLE
Docs: Add page and nav menu item for extensions

### LINKED ISSUE #3673: [Housekeeping] Add cookbook page for extensions
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the housekeeping request
Add a menu nav link and a cookbook page introducing https://extensions.kirby.design

### PR DESCRIPTION
**Pull request by RasmusKjeldgaard**
## Which issue does this PR close?

This PR closes #3673

## What is the new behavior?
See issue.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Reminders
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [x] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [x] Determine if your changes are a fix, feature or breaking-change, and add the matching label to your PR. If it is tooling, dependency updates or similar, add ignore-for-release.
- [x] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [x] Request that the changes are code-reviewed 
- [x] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

File: apps/cookbook/src/app/extensions/extensions-landing-page.component.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<article>
+  <h1>Kirby Extensions for Angular</h1>
+  <p>
+    The Kirby Extensions library is a collection of design guidelines and components that build on
```
Comment (Fuzzy3) [inline apps/cookbook/src/app/extensions/extensions-landing-page.component.html:4]: Does it also have 'guidelines' or is it actually "just" a collection of extra components?

File: apps/cookbook/src/app/extensions/extensions-landing-page.component.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    top of the core Kirby Design library.
+  </p>
+
+  <p>
+    Components in this library are developed and maintained by product teams, with support from
```
Comment (Fuzzy3) [inline apps/cookbook/src/app/extensions/extensions-landing-page.component.html]: What is meant by "product teams" - can anyone add components, also people from outside the world of bankdata?

File: apps/cookbook/src/app/extensions/extensions-landing-page.component.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    >
+      Team Kirby
+    </a>
+    .
+  </p>
+
+  <p>
+    The components follow the Kirby Design guidelines, but they differ from the core components by
+    being more product-specific. This means that components in this library are shared across one or
+    more products, and might not be universally useful to the whole ecosystem. Though some
```
Comment (Fuzzy3) [inline apps/cookbook/src/app/extensions/extensions-landing-page.component.html]:
```suggestion
    more products, and might not be universally useful to everyone. Though some
```

File: apps/cookbook/src/app/extensions/extensions-landing-page.component.html [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    components can also become core-candidates and graduate once they are battle-tested and initial
+    experimentation is over.
+  </p>
+
+  <p>
+    Extensions allow for quicker iteration and less focus on generalising behavior, while the test
+    and especially documentation burden is also significantly lower.
```
Comment (Fuzzy3) [inline apps/cookbook/src/app/extensions/extensions-landing-page.component.html:18]:
```suggestion
    and the amount of documentation is also significantly lower.
```

File: apps/cookbook/src/app/extensions/extensions-landing-page.component.html [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  </p>
+
+  <p>
+    The components are documented on
+    <a class="kirby-external-icon" href="http://extensions.kirby.design">extensions.kirby.design</a>
+    where additional information about installing, using and contributing extensions can be found.
```
Comment (Fuzzy3) [inline apps/cookbook/src/app/extensions/extensions-landing-page.component.html]:
```suggestion
    where additional information about installing, using and contributing to extensions can be found.
```

**Review by Fuzzy3 — CHANGES_REQUESTED** [review]
Consider rewriting some of the sentences so it is more general like the rest of the cookbook

File: apps/cookbook/src/app/extensions/extensions-landing-page.component.html (hunk 1 above)
Comment (RasmusKjeldgaard) [reply apps/cookbook/src/app/extensions/extensions-landing-page.component.html:4]: It is going to contain guidelines as well 👍

File: apps/cookbook/src/app/extensions/extensions-landing-page.component.html (hunk 4 above)
Comment (RasmusKjeldgaard) [reply apps/cookbook/src/app/extensions/extensions-landing-page.component.html:18]: This is not necessarily true, as the documentation can easily be more verbose. 
But the burden for creating this documentation is significantly lower, as it is very example-driven and more closely coupled to the components code.

Comment (Fuzzy3) [reply apps/cookbook/src/app/extensions/extensions-landing-page.component.html:18]: Aha that makes sense !

File: apps/cookbook/src/app/extensions/extensions-landing-page.component.html (hunk 1 above)
Comment (Fuzzy3) [reply apps/cookbook/src/app/extensions/extensions-landing-page.component.html:4]: Very nice !

File: apps/cookbook/src/app/extensions/extensions-landing-page.component.html (hunk 5 above)
Comment (RasmusKjeldgaard) [reply apps/cookbook/src/app/extensions/extensions-landing-page.component.html]: Hmm, this was intended to be a list of items (installing, using, contributing), so I don't think _to_ should be added here 🤔 

Reworded it instead, see 1634aa57a

File: apps/cookbook/src/app/extensions/extensions-landing-page.component.html (hunk 2 above)
Comment (RasmusKjeldgaard) [reply apps/cookbook/src/app/extensions/extensions-landing-page.component.html]: Yes, that is the intention. Reworded!

**Review by Fuzzy3 — APPROVED** [review]
Right in 👁️ !

**Pull request merged**